kmm/kmm-post-test-fixtures/src/commonMain/kotlin/com/gchristov/newsfeed/CalculateReadingTimeTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.gchristov.newsfeed
+
+import com.gchristov.newsfeed.kmmpostdata.util.ReadingTimeCalculator
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class CalculateReadingTimeTest {
```

Comment:
This test should be placed in the right place

Reply:
To clarify, this should probably go in the `kmm-post-data` test suite 👍

Reply:
Test is added and (that one) is passing